### PR TITLE
Bugfix for: #3106. Secborg modules boards are unlocked by research properly

### PR DESCRIPTION
**Pull request by DatBoiTim**
## About The Pull Request
#3106 Was broken. The node didn't unlock the upgrade board. So I fixed this.

## Why It's Good For The Game
Fixed an issue from a merged PR.

## Changelog
:cl: DatBoiTim
fix: SecBorg Module Upgrade now given by research.
/:cl:

### CODE REVIEW

**Comment by nikk735** [comment]
only printable at sec lathe = removed from game

**Comment by DatBoiTim** [comment]
> only printable at sec lathe = removed from game

And?

**Comment by nikk735** [comment]
removed content is bad just nerf it

**Comment by DatBoiTim** [comment]
> removed content is bad just nerf it

Fine, I guess I won't get too clandestine with this fix.

**Comment by bloons3** [comment]
If sec wants secborgs, they can work with Robotics to get them. Seems fine.

File: code/modules/research/techweb/all_nodes.dm [hunk 1]
```diff
@@ -363,10 +363,19 @@
 	display_name = "Cyborg Upgrades: Combat"
 	description = "Military grade upgrades for cyborgs."
 	prereq_ids = list("adv_robotics", "adv_engi" , "weaponry")
-	design_ids = list("borg_upgrade_vtec", "borg_upgrade_disablercooler")
+	design_ids = list("borg_upgrade_vtec", "borg_upgrade_disablercooler", )
```
Comment (Archanial) [inline code/modules/research/techweb/all_nodes.dm]:
```suggestion
	design_ids = list("borg_upgrade_vtec", "borg_upgrade_disablercooler")
```

**Comment by OceanFish1** [comment]
> If sec wants secborgs, they can work with Robotics to get them. Seems fine.

How often sec even wants to work with someone but their ego and a chaplain on cult rounds?

**Comment by DatBoiTim** [comment]
> > If sec wants secborgs, they can work with Robotics to get them. Seems fine.
> 
> 
> 
> How often sec even wants to work with someone but their ego and a chaplain on cult rounds? 

Main point of this is to just fix an error in original PR

File: code/modules/research/techweb/all_nodes.dm [hunk 2]
```diff
@@ -367,6 +367,15 @@
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 5000)
 	export_price = 5000
 
+/datum/techweb_node/cyborg_upg_security
+	id ="borg_upg_security"
+	description = "Allows Security to Make cyborgs into weapons of mass disabling."
```
Comment (ike709) [inline code/modules/research/techweb/all_nodes.dm]:
```suggestion
	description = "Allows Security to make cyborgs into weapons of mass disabling."
```

File: code/modules/research/techweb/all_nodes.dm [hunk 3]
```diff
@@ -367,6 +367,15 @@
...
+	display_name = "Cyborg Upgrade: Security"
```
Comment (ike709) [inline code/modules/research/techweb/all_nodes.dm]:
```suggestion
	display_name = "Cyborg Upgrades: Security"
```

**Comment by eeSPee** [comment]
Sec lathe doesn't make any sense. As a roboticist, you'd have to walk all the way to security to get the board, and walk back...

Why?

**Comment by MNarath1** [comment]
Behind advanced weaponary in its own node is way to far just make it working for normal adv weaponary node

**Comment by PowerfulBacon** [comment]
oh no departmental cooperation

File: code/modules/research/techweb/all_nodes.dm [hunk 4]
```diff
@@ -367,6 +367,15 @@
...
+	prereq_ids = list("cyborg_upg_combat", "mech_disabler")
+	design_ids = list("borg_transform_security")
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 10000)
```
Comment (PowerfulBacon) [inline code/modules/research/techweb/all_nodes.dm]: 10000 points for sec borgs

**Comment by badideaguy** [comment]
Power gaming is best cooperation, unless it is sec lathe, the sec lathe doesn't exist and any who say so are insane, sec lathe more like mindshields firing implant dispenser

File: code/modules/research/techweb/all_nodes.dm (hunk 4 above)
Comment (DatBoiTim) [reply code/modules/research/techweb/all_nodes.dm]: Ok I can tone it down to 5k

**Comment by DatBoiTim** [comment]
I'll nerf this in another PR. For now just get the fix in.

File: code/modules/research/techweb/all_nodes.dm (hunk 4 above)
Comment (DatBoiTim) [reply code/modules/research/techweb/all_nodes.dm]: Resolved. Resorting to pure bugfix

**Pull request merged**